snippets/comparison/managing head.md

```diff
@@ -37,14 +41,19 @@ export default function PostPage() {
 }
 ```
 
+The `generateMetadata` function can be used to fetch data and generate metadata based on the current segment dynamic params.
+
 ```jsx
-// app/posts/[id]/head.js
+// app/posts/[id]/page.js
 
-export default function Head({params}) {
-  return (
-    <>
-      <title>My post - {params.id}</title>
-    </>
-  )
+export async function generateMetadata({ params }) {
+  const post = await fetch(`https://.../${params.slug}`);
+  const data = await post.json();
+
+  return {
+    title: data.title,
+  };
 }
 ```
+
+You can see all the fields available for the metadata object in [here](https://beta.nextjs.org/docs/api-reference/metadata).
```

Comment:
Same here. Switching to something like "You can learn more about metadata in [the official docs for Metadata]"